docs/content/direct/observability.md

```diff
@@ -0,0 +1,69 @@
+# Observability in KubeStellar
+
+KubeStellar provides endpoints and integrations for observability and monitoring. This page describes the available observability features, how to access them, and how to use them in a typical deployment.
+
+## Metrics Endpoints
+
+KubeStellar controllers expose Prometheus-compatible metrics endpoints. These endpoints respond to HTTP requests for metrics and can be queried by any monitoring system; KubeStellar does not mandate how metrics are collected.
+
+### Metrics Endpoint Details
+
+- **Protocol:** HTTP
+- **Path:** `/metrics`
+- **Typical Port:** 8080 (sometimes 9090)
+- **Authentication/Authorization:**
+  - By default, metrics endpoints are not authenticated within the cluster. If you expose them outside the cluster, secure them appropriately.
+- **Example:**
+  - `http://<pod-ip>:8080/metrics`
+
+### Enabling Metrics
+
+KubeStellar controllers respond to requests for metrics at the `/metrics` endpoint by default. For one possible way to collect these metrics using Prometheus and Helm, see the [monitoring/README.md](https://github.com/kubestellar/kubestellar/tree/main/monitoring/README.md). You may use any monitoring stack you prefer.
```

Comment:
1. Helm plays no role in with collecting metrics.
2. That example also collects information from the debug endpoints.